Sources/TuistDependencies/SwiftPackageManager/Utils/SwiftPackageManagerGraphGenerator.swift

```diff
@@ -114,17 +114,10 @@ public final class SwiftPackageManagerGraphGenerator: SwiftPackageManagerGraphGe
 
         let packageToProject = Dictionary(uniqueKeysWithValues: packageInfos.map { ($0.name, $0.folder) })
         let packageInfoDictionary = Dictionary(uniqueKeysWithValues: packageInfos.map { ($0.name, $0.info) })
-        let targetDependencyToFramework: [String: Path] = packageInfos.reduce(into: [:]) { result, packageInfo in
-            let artifactsFolder = artifactsFolder.appending(component: packageInfo.name)
-            packageInfo.info.targets.forEach { target in
-                guard target.type == .binary else { return }
-                result[target.name] = Path(artifactsFolder.appending(component: "\(target.name).xcframework").pathString)
-            }
-        }
         let (targetToProducts, targetToResolvedDependencies) = try packageInfoMapper.preprocess(
             packageInfos: packageInfoDictionary,
             productToPackage: productToPackage,
-            targetDependencyToFramework: targetDependencyToFramework
+            artifactsFolder: artifactsFolder
```

Comment:
I think the artifactsFolder can be removed from the:
```
        let packageInfos: [(name: String, folder: AbsolutePath, artifactsFolder: AbsolutePath, info: PackageInfo)]
```
as it is no longer used

Reply:
it is used: let artifactsFolderForPackage = artifactsFolder.appending(component: packageInfo.key)